app/elements/pw-shell.html

```diff
@@ -250,13 +286,44 @@
 
       // Save the initial URL so that we don't double-render the first load.
       this._currentPageURL = this._sectionRoute.path;
+
+      // Save which version of docs we're looking at, since we need to
+      // maintain this state across the app. Note that not all sections are versioned.
+      if (this._sectionData.version === '1.0' || this._sectionData.version === '2.0')
+        this.activeDocsVersion = this._sectionData.version;
+    },
+
+    _listboxVersionChanged: function(event) {
+      var newSelection = event.detail.item.textContent;
+      if (newSelection === this.activeDocsVersion)
+        return;
+      this.activeDocsVersion = newSelection;
+      this.set('_sectionRoute.path', '/' + this.activeDocsVersion + '/start/');
     },
 
     _toggleDrawer: function() {
+      // Sync the version selector with the active docs version.
+      this.$.versionSelect.selected = this.activeDocsVersion == '1.0' ? 0 : 1;
```

Comment:
True, but this means you'll be doing extra work every time you toggle the drawer (e.g. navigating between 2.0 pages, or when closing the drawer and the listbox goes out of view). There's also cases where you can change the `activeDocsVersion` without toggling the drawer (e.g. switch versions, open the drawer, use the browser back button to switch back to the last version). Either way, it's more "correct" to update the listbox when the version has actually changed (not just because the drawer is toggled).